build/RunTestsOnHelix.cmd

```diff
@@ -38,6 +37,7 @@ REM We downloaded a special zip of files to the .nuget folder so add that as a s
 dotnet nuget list source --configfile %TestExecutionDirectory%\nuget.config
 PowerShell -ExecutionPolicy ByPass "dotnet nuget locals all -l | ForEach-Object { $_.Split(' ')[1]} | Where-Object{$_ -like '*cache'} | Get-ChildItem -Recurse -File -Filter '*.dat' | Measure"
 dotnet nuget add source %DOTNET_ROOT%\.nuget --configfile %TestExecutionDirectory%\nuget.config
+if exist %TestExecutionDirectory%\Testpackages dotnet nuget add source %TestExecutionDirectory%\Testpackages --name testpackages --configfile %TestExecutionDirectory%\nuget.config
```

Comment:
Does the script work as is for local helix execution or are changes needed to the instructions here: https://github.com/dotnet/sdk/blob/main/documentation/project-docs/repro-helix-failure.md

We don't do this often but that means it's fragile to changes if we're not careful.

Reply:
Didn't know about this possibility to repro helix locally, thanks.

I've followed the instructions and they seem to work, in particular the nuget source is added correctly:

```
C:\temp\helix\payload-dir>if exist C:\Users\janjones\AppData\Local\Temp\dotnetSdkTests\jxi5tacr.5hi\Testpackages dotnet nuget add source C:\Users\janjones\AppData\Local\Temp\dotnetSdkTests\jxi5tacr.5hi\Testpackages --name testpackages --configfile C:\Users\janjones\AppData\Local\Temp\dotnetSdkTests\jxi5tacr.5hi\nuget.config
Package source with Name: testpackages added successfully.
```